CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## UNRELEASED
+
+### Fixes
+
+* Updates the docs `beforeInsert` handler to avoid ending with different modes being set between `_id`, `aposLocale` and `aposMode`.
+* Adds a migration to fix potential corrupted data have different modes set between `_id`, `aposLocale` and `aposMode`.
```

Comment:
have -> having
have different -> with different

Reply:
changes `have different` but did not find the first `have`